addon/components/paper-autocomplete.js

```diff
@@ -217,7 +218,7 @@ export default Ember.Component.extend(HasBlockMixin, {
   shouldHide: Ember.computed.not('isMinLengthMet'),
 
   isMinLengthMet: Ember.computed('searchText', 'minLength', function() {
-    return this.get('searchText').length >= this.get('minLength');
+    return this.get('searchText.length') >= this.get('minLength');
```

Comment:
Does this fix a bug because `searchText` is sometimes not a string? We are trying to isolate changes to autocomplete to minimize merge conflicts, since there is already WIP in another PR.